web/ui/react-app/src/pages/graph/ExpressionInput.tsx

```diff
@@ -117,8 +117,14 @@ const ExpressionInput: FC<CMExpressionInputProps> = ({
           queryHistory
         ),
       });
+
+    let highlighter = syntaxHighlighting(promqlHighlighter);
```

Comment:
Rather than needlessly calling this function twice when in dark mode, how about:

```suggestion
    let highlighter = syntaxHighlighting(theme === 'dark' ? darkPromqlHighlighter : promqlHighlighter);
```